ark-testing/src/bark.rs

```diff
@@ -30,13 +30,8 @@ impl BarkCommand {
 					args: vec![]}
 				)
 			},
-			Err(env::VarError::NotPresent) => {
-				Ok(BarkCommand {
-					exe_path: which("cargo")?,
-					args: ["run", "--package", "bark-client", "--"].iter().map(|x| x.to_string()).collect()
-				})
-			},
-			Err(_) => bail!("Failed to read BARK_EXEC"),
+			Err(env::VarError::NotPresent) => panic!("BARK_EXEC is not set"),
+			Err(_) => anyhow::bail!("Failed to read BARK_EXEC"),
```

Comment:
nit: can't this become a simple `.expect` at this point? 

btw it seems to be doing a panic in one case and error in the other, probably better streamline, I think for tests, panicking is probably ok. with a expect

Reply:
I see you went for bail then. Fine by me :)

Reply:
Oh wait, you didn't shorten them, though. If you want to just have an error, not a panic, you can just do ".context("can't access XXX_EXEC or not present")?` instead of the match. I don't think there is too much value in distinguishing the variants.
But well, not a blocker tbh.